de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/ASwtBotTestCase.java

```diff
@@ -32,9 +32,13 @@
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.rules.TestName;
+<<<<<<< HEAD
+import org.junit.rules.Timeout;
+=======
```

Comment:
Weird merge remains.

Reply:
Fixed. Did not pay attention when merging.